src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -0,0 +1,46 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+//@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity(name = "Users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    @NonNull
+    @Column(name="UserID", unique = true)
+    private Long userId;
+
+    @Column(name = "UserName")
+    private String userName;
+
+    @NonNull
+    @Column(name = "Status")
+    private int status;
+
+    @NonNull
+    @Column(name = "UserKey")
+    private long userKey;
+
+    @OneToMany (
+            targetEntity =  Order.class,
+            mappedBy = "buyerId",
+            fetch = FetchType.LAZY
```

Comment:
W przypadku relacji `@OneToMany` domyślna wartość atrybutu `fetch` to właśnie `LAZY`, więc... j.w. :)